qemu/docker-qemu.sh

```diff
@@ -55,5 +55,5 @@ docker run ${ARGS[@]} \
     --net=host \
     --privileged \
     --rm \
-    -it -v $(pwd):/tmp -v /dev:/dev \
```

Comment:
Don't forget to reflect the change in README: specify `/dev:/dev` instead of `current directory:/tmp`.